schemas/namespace/identity-namespace.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/identity-namespace",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Identity Namespace",
+  "type": "object",
+  "description": "Identity Namespace Schema.",
+  "definitions": {
+    "identity-namespace": {
+      "properties": {
+        "xdm:id": {
+          "title": "Unique Namespace Id",
+          "type": "string",
+          "description": "Unique Identifier for namespace"
+        },
+        "namespaceName": {
+          "title": "Namespace Name",
+          "type": "string",
+          "description": "This is human readable name for the namespace"
+        },
+        "xdm:namespaceCode": {
+          "title": "Namespace Code",
+          "type": "string",
+          "description": "This code is a human readable identifier for the namespace"
+        },
+        "description": {
+          "title": "Namespace Description",
+          "type": "string",
+          "description": "Description for the namespace"
+        },
+        "idType": {
```

Comment:
you're missing the `xdm` here

Also, don't understand what this means...

Reply:
It represents a type for Namespace. Example "foo@bar.com" could of type "Email". You could call it a category of namespaces.

Reply:
but that's not a namespace - at least in the XDM terminology.  And that is causing the confusion.

Is there an alternate term, for example, identity type?

Reply:
ya we could say that. @harikb -- any recommendations here

Reply:
if we use "xdm:code" or "xdm:name", will it have conflicts because i see there is a datatype defined for "xdm:name" already ("/xdm/common/extensible#/definitions/@context") and probably for other fields as well. Would it makes sense to have our own namespace and define fields using it ?

Reply:
as long as the type is the same (in this case, they are both strings) - you are fine.